src/libs/tmdk/tests/init.test.ts

```diff
@@ -1,22 +1,59 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
```

Comment:
why?

Reply:
I removed the code that required the rule to be disabled